pom.xml

```diff
@@ -1,36 +1,132 @@
-<?xml version="1.0"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-	<modelVersion>4.0.0</modelVersion>
-	<parent>
-		<groupId>se.sundsvall.dept44</groupId>
-		<artifactId>dept44-service-parent</artifactId>
-		<version>2.1.3</version>
-		<relativePath /> <!-- lookup parent from repository -->
-	</parent>
-	<groupId>se.sundsvall</groupId>
-	<artifactId>template-spring-boot</artifactId>
-	<version>1.0</version>
-	<name>template</name>
-	<properties>
-		<!-- Pipeline properties -->
-		<docker.image.name>ms-template</docker.image.name>
-		<exposed.app.port>8080</exposed.app.port>
-		<published.container.port.test>xxx</published.container.port.test>
-		<published.container.port.production>xxx</published.container.port.production>
-	</properties>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-devtools</artifactId>
-			<scope>runtime</scope>
-			<optional>true</optional>
-		</dependency>
-		
-		<!-- Test -->
-		<dependency>
-			<groupId>se.sundsvall.dept44</groupId>
-			<artifactId>dept44-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>se.sundsvall.dept44</groupId>
+        <artifactId>dept44-service-parent</artifactId>
+        <version>3.2</version>
```

Comment:
Update dependency as latest version is now 3.4

Reply:
Now Updated